generate/generate.go

```diff
@@ -1291,7 +1291,7 @@ func (s *service) generateConvertCode(cmd, name, typ string) {
 	pn := s.pn
 
 	switch typ {
-	case "string":
+	case "string", "UUID":
```

Comment:
should it be UUID or uuid?

Reply:
We are explicitly setting it to `"UUID"` in L2096. As of now the value is `UUID` only for `managementserverid`. And `uuid` is being treated as a string. Changing this to `uuid` causes a lot of changes at other places and removes some helper functions.
https://github.com/apache/cloudstack-go/pull/60/files#diff-39a24c8d3907e38beb5e39b5f0a40df7aa73c38449615f140ec3fca502900e86L2096

This seems to be there because of [this](https://github.com/apache/cloudstack-go/pull/21).

Reply:
Great thanks for checking @vishesh92